extension/json/test/tinysnb_all_shortest_path_json.test

```diff
@@ -1,4 +1,4 @@
--DATASET CSV tinysnb_json
+-DATASET JSON tinysnb_json
```

Comment:
Remove the tinysnb json dataset, use the conversion function instead.

Reply:
I wouldn't do this. `tinysnb_json` covers cases like unstructured or missing fields, which isn't covered by conversion.